salt/orch/kubernetes.sls

```diff
@@ -66,9 +66,15 @@ etcd_discovery_setup:
     - sls:
       - etcd-discovery
     - require:
-      - salt: ca_setup
```

Comment:
No, that's why I removed it, if you check `etcd-discovery` this targets the etcd instance on the admin node with `http` for discovery purposes only: https://github.com/kubic-project/salt/blob/master/salt/etcd-discovery/init.sls#L3-L4.

I removed it since it's not a real requirement.